package.json

```diff
@@ -40,9 +40,9 @@
     "eslint-plugin-unicorn": "^19.0.1",
     "eslint-plugin-vue": "^6.0.0",
     "jest": "^26.0.0",
-    "jest-preset-stylelint": "^3.0.0",
+    "jest-preset-stylelint": "^4.2.0",
     "lerna": "^3.16.4",
-    "stylelint": "^11.1.0",
+    "stylelint": "14.0.0",
```

Comment:
Maybe `^14`?

Reply:
By pinning to v14, we ensure that this package is compatible with the entire major v14 range of `stylelint` as this package will test using that specific version of stylelint.

e.g. A new feature or rule is introduced in v14.5 of `stylelint` and this package adds it, this package is no longer compatible with earlier versions of `stylelint`. Unless this new rule's compatibility can be detected and progressively added when linked to the correct stylelint version, but I cannot find such feature in the stylelint documentation.

I choose this route because:
1. The peerDependency on `stylelint` has previously been relaxed (#682)
2. To ensure compatibility with all v14 stylelint versions
3. This package is still in major version zero and therefore can have breaking changes each new release until v1 comes out, according to [SemVer](https://semver.org/#spec-item-4)

I see 2 options:
- Pin `stylelint` devDependency to ensure compatibility across the entire v14 version range. 
  - Compatibility with consumer's installed stylelint version is handled through whether stylelint throws an error.
- Set peerDependencies starting from this update to `^14.0.0`
  - When adding new stylelint features, there must also be checked if the correct peerDependency is set.
  - Compatibility check is handled through the package manager.

`stylelint` advises usage of peerDepencencies. https://stylelint.io/developer-guide/plugins#peer-dependencies

What are your thoughts?

Reply:
> By pinning to v14, we ensure that this package is compatible with the entire major v14 range of stylelint as this package will test using that specific version of stylelint.

I prefer to have the same version as we do in `peerDependencies`. It feels like keeping them in sync does help more then cause problems.